hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/regionserver/TestCompactionTool.java

```diff
@@ -65,6 +79,7 @@ public void setUp() throws Exception {
   public void tearDown() throws Exception {
     this.testUtil.shutdownMiniCluster();
     testUtil.cleanupTestDir();
+    ChunkCreator.instance = null;
```

Comment:
Will this cause memory leak? And since we will always reset USEMSLAB_KEY to false, do we really need to do this?

Reply:
Setting it to null after shutdown of hbase cluster in test, so that it is not available for next test case in the file. Memory blocks of ChunkCreator should be collected by GC as cluster is shutdown.